

class ContinueFromLoopException(Exception):
    pass

class LastIterationException(Exception):
    pass